downstream/modules/eda/proc-eda-delete-rulebook-activations.adoc

```diff
@@ -0,0 +1,8 @@
+[id="proc-eda-delete-rulebook-activations"]
+
+== Deleting rulebook activations
+
```

Comment:
Suggest "Delete rulebook activations" as a subtask of the overall Activating webhook rulebooks. See above.